native/src/components/MapView.tsx

```diff
@@ -146,28 +150,30 @@ const MapView = ({
     setSheetSnapPointIndex(1)
   }
 
+  const deactivateFollowUserLocation = () => {
+    if (followUserLocation) {
+      setFollowUserLocation(false)
+    }
+  }
```

Comment:
Why not just, setting it to false if it already is false does not have an effect.
```suggestion
  const deactivateFollowUserLocation = () => setFollowUserLocation(false)
```